utility/types/message.go

```diff
@@ -180,6 +185,7 @@ func validateRelayChains(chains []string) Error {
 }
 
 func getMessageType(msg Message) string {
+	// GetMessageName() is a method on the Message interface, so we can't use it here.
```

Comment:
If you really think this comment is necessary, I'd suggest renaming this `getMessageType` to something like `getProtoDescriptorName` so that the intent is clear